### ISSUE TITLE
Log func test cleanup errors

### ISSUE THREAD

**Issue by rebstar6** [vinyldns/vinyldns#280]
In cleanup for most of the func tests, we `except: pass` on cleanup failures. this can cause other tests to fail because of conflicting data. We should log the error returned before passing on that cleanup.

**Comment by pauljamescleary** [comment]
Makes sense, low hanging fruit and should be easy for someone to knock out

**Comment by nspadaccino** [comment]
Func tests have been overhauled in https://github.com/vinyldns/vinyldns/pull/1058, this issue has been resolved
